ui/store/actions.ts

```diff
@@ -5451,6 +5451,30 @@ export async function tokenRatesStopPollingByPollingToken(
   await removePollingTokenFromAppState(pollingToken);
 }
 
+/**
+ * Informs the DeFiPositionsController that the UI requires defi positions polling
+ *
+ * @returns polling token that can be used to stop polling.
+ */
+export async function deFiStartPolling(): Promise<string> {
+  const pollingToken = await submitRequestToBackground('deFiStartPolling', [
+    null,
+  ]);
+
+  await addPollingTokenToAppState(pollingToken);
+  return pollingToken;
+}
+
+/**
+ * Informs the DeFiPositionsController that the UI no longer needs defi positions polling
+ *
+ * @param pollingToken - Poll token received from calling deFiStartPolling
+ */
+export async function deFiStopPolling(pollingToken: string) {
+  await submitRequestToBackground('deFiStopPolling', [pollingToken]);
+  await removePollingTokenFromAppState(pollingToken);
+}
+
```

Comment:
I would really prefer that we not add new functions to `ui/store/actions.ts`, it's already over 6,000 lines, and we're trying to reduce the size.

Does it make sense to put these in `ui/hooks/useDeFiPolling.ts`?